Makefile

```diff
@@ -36,12 +36,11 @@ help:
 	@echo "--------------------"
 	@echo ""
 	@echo "i18n-extract: extract all strings from application (both front- and back-end)"
-	@echo "i18n-crowdin-upload branch=<crowdin-branch>: upload strings to Crowdin"
-	@echo "i18n-crowdin-download branch=<crowdin-branch>: download strings from Crowdin and compile"
-	@echo "i18n-crowdin-stats branch=<crowdin-branch>: output information about translation status"
+	@echo "i18n-upload branch=<crowdin-branch>: upload sources to Crowdin"
+	@echo "i18n-download branch=<crowdin-branch>: download strings from Crowdin and regenerate files"
+	@echo "i18n-stats branch=<crowdin-branch>: output information about translation status"
 	@echo "i18n-django-compilemessages: compiles .po files to .mo files for Django"
 	@echo "i18n-install-font name=<noto-font>: Downloads and installs a new or updated font"
-	@echo "i18n-generate-fonts: Generates custom fonts and CSS based on current languages and app strings"
```

Comment:
I liked having this separately :)

Reply:
Could it just depend on the `i18n-crowdin-download` recipe instead?

Reply:
Alternatively, as mentioned above, we could make a `i18n-download` recipe that just depends on `i18-crowdin-download` + `i18-generate-fonts`..